CMBLensingSPT3GInterface/src/CMBLensingSPT3GInterface.jl

```diff
@@ -208,15 +223,15 @@ end
 """
 Convert a CMBLensing FlatS02 to a 3G MapSpectraTEB.
 """
-function MapSpectraTEB(f)
-    frame = Frame(Fourier(f), "TEB")
+function MapSpectraTEB(f::FlatIEBFourier)
+    frame = Frame(f, "TEB")
```

Comment:
Is there a reason to restrict a user from going from _any_ `FlatS02` to a python `MapSpectraTEB` in one step? Calling e.g. `MapSpectraTEB(f::FlatIQUMap)`(which worked before) is convenient